.claude/commands/pr-unresolved.md

```
@@ -61,32 +74,92 @@ When the user runs this command with a PR number:
-     - File path
-     - Line number
-     - Full comment body from the thread
+     - Thread number (for reference)
+     - Thread ID (starts with `PRRT_`) - used for resolving
+     - Comment ID / databaseId (numeric) - used for posting replies
+     - File path and line number
+     - First 50-100 chars of comment body as topic summary
      - Whether it's outdated or collapsed
 
+   **Example format:**
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)